### ISSUE TITLE
Writing obspy catalog objects to QuakeML skips magnitude error

### ISSUE THREAD

**Issue by komjongshawn** [obspy/obspy#1414]
Hello,

I am trying to compute magnitude errors for events in a catalog that is stored in QuakeML format using obspy.  I can add an AttribDict for the errors to an obspy Catalog object and manipulate it while it is in the Catalog object; however, the errors do not get written out to the QuakeML file when I save it.

I am using obspy 1.0.0 with Python 2.7 in Anaconda on an Ubuntu 14.04 system.

Sample code:
```Python
import obspy

#Create an obspy catalog object, add an event to it, and assign a 
#magnitude
qml = obspy.core.event.Catalog()
qml.events.append(obspy.core.event.Event())
qml[0].magnitudes.append(obspy.core.event.magnitude.Magnitude())
qml[0].magnitudes[0].mag = 1.0
qml[0].magnitudes[0].magnitude_type = "ML"

#Append a magnitude error
mag_err = obspy.core.util.AttribDict()
mag_err["error"] = 0.2
qml[0].magnitudes[0].mag_errors = mag_err

#Write the object to a QuakeML
qml.write("test.xml", "quakeml")

#Read the resultant QuakeML file
result = obspy.core.quakeml.readQuakeML("test.xml")

#Try and retrieve the error that was stored in both the original and
#read in objects
print(qml[0].magnitudes[0].mag_errors["error"])
print(result[0].magnitudes[0].mag_errors["error"])
```

Shawn

**Comment by megies** [comment]
The problem is that you're not using the correct key name.. the "..._errors" expected keys are: `uncertainty` and/or `lower_uncertainty`&`upper_uncertainty` and `confidence_level`. The documentation on the "..._errors" fields is really bad, I'll fix it with #1418 and also will issue a warning when an unexpected attribute is set.

So in your code you should replace..
```python
mag_err["error"] = 0.2
```
with..
```python
mag_err["uncerteinty"] = 0.2
```
To make it a bit shorter you can also set the errors attribute with a plain dictionary, it will be converted to a `QuantityError` automatically:
```python
qml[0].magnitudes[0].mag_errors = {"uncertainty": 0.2}
```

**Comment by megies** [comment]
Should be clear now and handled much better with #1418 and #1426. Please reopen if anything is still unclear.
